lib/src/components/NxTag/NxTag.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+import { faPlusCircle } from '@fortawesome/free-solid-svg-icons';
+
+import './NxTag.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTag: FunctionComponent<Props> =
```

Comment:
You're using the same `Props` and `propTypes` on both `NxTag` and `NxSelectableTag`, which suggests that `NxTag` accepts the `tagSelected` and `onTagSelected` props when it does not.  You need separate types for these two components.

Reply:
> You're using the same Props and propTypes on both NxTag and NxSelectableTag, which suggests that NxTag accepts the tagSelected and onTagSelected props when it does not. You need separate types for these two components.

FYI I've had to address this in my branch because of an issue around `onSelect` conflicting with a native prop